packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/middleware/plugins/multisite.ts

```diff
@@ -28,12 +28,24 @@ class MultisitePlugin implements MiddlewarePlugin {
       siteResolver,
       // This function allows resolving site from sc_site cookie, which could be useful in case of Vercel preview URLs. Accepts NextRequest.
       useCookieResolution: () => process.env.VERCEL_ENV === 'preview',
+      // Тhe attributes of the site cookie with recommended values
+      siteCookieAttributes: {
+        secure: true,
+        expires: this.getExpirationDate(10),
+        httpOnly: true,
+        sameSite: 'lax',
+      },
```

Comment:
Should these be configurable? What do you think?

Reply:
No, I don't think they should be ... See https://github.com/Sitecore/jss/pull/1746#discussion_r1497872002